mediapipe/examples/ios/common/CommonViewController.mm

```diff
@@ -113,7 +113,7 @@ - (void)viewWillAppear:(BOOL)animated {
       self.videoSource = [[MPPPlayerInputSource alloc] initWithAVAsset:video];
       [self.videoSource setDelegate:self queue:self.videoQueue];
       dispatch_async(self.videoQueue, ^{
-        [self.videoSource start];
+        [self startGraphAndVideo];
```

Comment:
Can you align this with "startGraphAndCamera" and move the dispatch block to the new function? Thanks